### PR TITLE
Add more information on individual controllers

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/reference/kube-controllers/configuration.md [hunk 1]
```diff
@@ -55,3 +55,42 @@ The following environment variables can be used to configure the {{site.prodname
...
+### Workload endpoint controller
+
+The workload endpoint controller automatically syncs Kubernetes pod label metadata changes to the Calico data store by updating
+the corresponding workload endpoints appropriately.
+
+The workload endpoint controller is enabled by default.
+
+### Profile controller
+
+The profile controller syncs Kubernetes namespace label changes to the Calico data store.
```
Comment (emanic) [inline master/reference/kube-controllers/configuration.md]: `Calico` --> `{{site.prodname}}`

File: master/reference/kube-controllers/configuration.md [hunk 2]
```diff
@@ -55,3 +55,42 @@ The following environment variables can be used to configure the {{site.prodname
...
 [in-cluster-config]: https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
 [kubeconfig]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
+
+## About each controller
+
+### Node controller
+
+The node controller automatically cleans up configuration for nodes that no longer exist.
+
+The node controller is not enabled by default. However, the Calico self-hosted manifests do enable this controller.
```
Comment (emanic) [inline master/reference/kube-controllers/configuration.md]: On  line 19, it says all four are enabled by default.

Comment (emanic) [reply master/reference/kube-controllers/configuration.md]: Also `Calico` --> `{{site.prodname}}`

File: master/reference/kube-controllers/configuration.md [hunk 3]
```diff
@@ -55,3 +55,42 @@ The following environment variables can be used to configure the {{site.prodname
...
+
+To enable the node controller, perform the following two steps.
+
+1. Add "node" to the list of enabled controllers in the environment for kube-controllers. For example: `ENABLED_CONTROLLERS=workloadendpoint,profile,policy,node`
+1. Configure calico/node with a Kubernetes node reference by adding the following snippet to the environment section of the calico-node daemon set.
```
Comment (emanic) [inline master/reference/kube-controllers/configuration.md]: `calico/node` --> `{{site.nodecontainer}}`
the other one can be `calico-node` --> `{{site.noderunning}}`

File: master/reference/kube-controllers/configuration.md [hunk 4]
```diff
@@ -55,3 +55,42 @@ The following environment variables can be used to configure the {{site.prodname
...
+```
+- name: CALICO_K8S_NODE_REF
+  valueFrom:
+    fieldRef:
+      fieldPath: spec.nodeName
+```
+
+### Policy controller
+
+The policy controller syncs Kubernetes network policies to the Calico data store.
```
Comment (emanic) [inline master/reference/kube-controllers/configuration.md]: `Calico` --> `{{site.prodname}}`

**Comment by caseydavenport** [comment]
@emanic ready for another look

**Pull request merged**